src/Panels/ChatLogPanel.cs

```diff
@@ -375,12 +375,13 @@ public void WelcomeChatMessage()
         }
 
         /// <summary>
-        ///     Prints a game message to the ChatLogPanel with MessageType.NORMAL.
+        ///     Prints a game message to the ChatLogPanel & Chirper with MessageType.NORMAL.
```

Comment:
I don't think that both should be done _though I have no comment on which should be excluded or defaulted to (if a setting is implemented)_.

Reply:
Yeah, perhaps we can get a third opinion from one of the main devs on this.